### ISSUE TITLE
Got attribute error when import tensorflow

### ISSUE THREAD

**Issue by vagitablebirdcode** [rsalmei/alive-progress#207]
When I import tensorflow which has its python plugine `logging`, I got an attribute error after using the alive_it. The output contents are as follow:
```python
/lib/python3.9/site-packages/alive_progress/core/hook_manager.py", line 134, in _set_stream
    return handler.setStream(stream)
/lib/python3.9/logging/__init__.py", line 1108, in setStream
    self.stream = stream
AttributeError: can't set attribute
```
My shell and terminal can't flush the cursor and words When I run this programme unless I restart it.

**Comment by xiachenrui** [comment]
I meet same problem when use debug mode in VScode

**Comment by rsalmei** [comment]
This is the same as #217, and it is fixed in 3.0.1, enjoy!
